template_config.yml

```diff
@@ -61,9 +61,11 @@ pulp_settings:
   - /tmp
   allowed_import_paths:
   - /tmp
```

Comment:
I don't know what "test flatpack index under pulp_settings" means.

Related to modifications in this file, there are apparently five CI jobs that could either run or skip `pulp_container/tests/functional/api/test_flatpak.py::test_flatpak_install`, depending on whether or not `flatpak_index: true` is enabled for them in some way: "test (pulp)", "test (azure)", "test (s3)", "test (stream)", and "test (lowerbounds)".  The comment at <https://github.com/pulp/pulp_container/blob/c384e8a3ddb22258c3cc70760b8520322560e57c/template_config.yml#L67> argues that "test (s3)" should skip it for now.  And "test (azure)" currently runs the test.

IIRC, the original idea (discussed outside this PR, IIRC) was to enable this for just a subset of CI jobs, to see that both variants (enabled/disabled) work.  But I'm just as fine with enabling it for all jobs but "test (s3)".  So:

(1) For which of the four remaining jobs "test (pulp)", "test (azure)", "test (stream)", and "test (lowerbounds)" shall it be enabled?
(2) If it shall be enabled for "test (pulp)", what modifications would it take to do so?
(3) If it shall be enabled for "teast (lowerbounds)", what modifications would it take to do so?  Add some
```
pulp_settings_lowerbounds:
  flatpak_index: true
```
section to `template_config.yml`?

Reply:
Leave it enabled for azure and pulp.

```
pulp_settings:
  flatpak_index: true

...

pulp_settings_azure:
   flatpak_index: true
```

Reply:
done in 30c187d5019a5ad33c4c2781ee209985d5940cae

(I had feared that `pulp_settings` might be some kind of base case, and modifying it might also affect all the other `pulp_settings_*`.)

Reply:
> (I had feared that `pulp_settings` might be some kind of base case, and modifying it might also affect all the other `pulp_settings_*`.)

So it looks like my fears were not unfounded, and at <https://github.com/pulp/pulp_container/actions/runs/5623208588/job/15237438136?pr=1301> "test (s3)" now started to execute `pulp_container/tests/functional/api/test_flatpak.py::test_flatpak_install` again (and, of course, failed).

Reply:
So with the latest changes in f08a44fd1f71030874bad60257d4bff4a3c9373f, it runs the test in "test (pulp)", "test (azure)", and "test (lowerbounds)", and skips it in "test (s3)" and "test (stream), and all are green.